Layers/Renderers/Building/BuildingRenderer.gd

```diff
@@ -1,84 +1,126 @@
 extends FeatureLayerCompositionRenderer
 
-
 var building_base_scene = preload("res://Buildings/BuildingBase.tscn")
-var plain_walls_scene = preload("res://Buildings/Components/PlainWalls.tscn")
 var flat_roof_scene = preload("res://Buildings/Components/FlatRoof.tscn")
 var pointed_roof_scene = preload("res://Buildings/Components/PointedRoof.tscn")
 
+var fallback_wall = preload("res://Resources/Textures/Buildings/PlainWallResources/Industrial.tres")
+
+var wall_resources = [
+	# "apartments": 0
+	preload("res://Resources/Textures/Buildings/PlainWallResources/House.tres"),
+	# "house": 1
+	preload("res://Resources/Textures/Buildings/PlainWallResources/House.tres"),
+	# "shack": 2
+	preload("res://Resources/Textures/Buildings/PlainWallResources/House.tres"),
+	# "industrial": 3
+	preload("res://Resources/Textures/Buildings/PlainWallResources/Industrial.tres"),
+	# "office": 4
+	preload("res://Resources/Textures/Buildings/PlainWallResources/Office.tres"),
+	# "supermarket": 5
+	preload("res://Resources/Textures/Buildings/PlainWallResources/House.tres"),
+	# "retail_restaurant": 6
+	preload("res://Resources/Textures/Buildings/PlainWallResources/House.tres"),
+	# "historic": 7
+	preload("res://Resources/Textures/Buildings/PlainWallResources/House.tres"),
+	# "religious": 8
+	preload("res://Resources/Textures/Buildings/PlainWallResources/House.tres"),
+	# "greenhouse": 9
+	preload("res://Resources/Textures/Buildings/PlainWallResources/House.tres"),
+	# "concrete": 10
+	preload("res://Resources/Textures/Buildings/PlainWallResources/House.tres"),
+]
+
 var floor_height = 2.5 # Height of one building floor for calculating the number of floors from the height
-var fallback_height = 10
+var fallback_height = 20
```

Comment:
Isn't 20m quite high for a fallback height?
> Ein durchschnittliches Einfamilienhaus erreicht eine Höhe von etwa 10 Metern, bestehend aus der Höhe von Erdgeschoss und Obergeschoss (je 2,60 Meter), Decken, Bodenplatte und Dach (etwa 3 Meter). Bauvorschriften limitieren die maximale Höhe regional.
https://www.hausjournal.net/einfamilienhaus-hoehe